dask_expr/array/core.py

```diff
@@ -0,0 +1,215 @@
+import operator
+from typing import Union
+
+import dask.array as da
+from dask.base import DaskMethodsMixin, named_schedulers
+from dask.utils import cached_cumsum, cached_property
+from toolz import reduce
+
+from dask_expr import _core as core
+
+T_IntOrNaN = Union[int, float]  # Should be Union[int, Literal[np.nan]]
+
+
+class Array(core.Expr, DaskMethodsMixin):
+    _cached_keys = None
+
+    __dask_scheduler__ = staticmethod(
+        named_schedulers.get("threads", named_schedulers["sync"])
+    )
+    __dask_optimize__ = staticmethod(lambda dsk, keys, **kwargs: dsk)
+
+    def __dask_postcompute__(self):
+        return da.core.finalize, ()
+
+    def __dask_postpersist__(self):
+        return FromGraph, (self._meta, self.chunks, self._name)
```

Comment:
```suggestion
        return FromGraph, (self._meta, self.chunks, self._name)
        
    def __array__(self):
        return self.simplify().compute()

```

Reply:
> Mind adding an automatic .simplify() when calling .compute()? That will help some experiments with Xarray

Yeah, I've pushed something very similar to this.